aibolit/model/model.py

```diff
@@ -82,7 +82,20 @@ class PatternRankingModel(BaseEstimator):
     def __init__(self):
         self.do_rename_columns = False
         self.model = None
-        self.features_conf = None
+        self._features_conf = None
+
+    @property
+    def features_conf(self) -> dict[Any, Any]:
+        if self._features_conf is not None:
+            return self._features_conf
+        raise RuntimeError(
+            'features_conf is not available, '
+            'probably need to run fit_regressor first',
+        )
+
+    @features_conf.setter
+    def features_conf(self, value: dict[Any, Any]) -> None:
+        self._features_conf = value
```

Comment:
Ironically, I had to fix `features_conf` being either `None` or not using a setter. But I do not see another way of doing it with the present design, since `features_conf` gets set in several places, including tests (may want to change tests later).

Reply:
Reverted this change as apparently it broke integration tests.